superdesk/archive_async/service.py

```diff
@@ -290,7 +290,7 @@ async def on_created(self, docs: list[ArchiveResourceModel]) -> None:
             else:
                 await app.on_archive_item_updated({"task": doc.task}, doc, ItemOperation.CREATE)
 
-        await get_resource_service("content_types").set_used(profiles)
+        get_resource_service("content_types").set_used(profiles)
```

Comment:
[set_used](https://github.com/superdesk/superdesk-core/blob/42884b9af70f5b9b02fa1a6597129aeecc91e7a2/apps/content_types/content_types.py#L243C14-L243C29) can actually be updated to async as well, by using [find_and_modify_async](https://github.com/superdesk/superdesk-core/blob/async/superdesk/eve_async/service.py#L125) instead